src/test/mochitest/examples/math.min.js

```diff
@@ -1,27 +1,3 @@
-function add(a, b, k) {
-  var result = a + b;
-  return k(result);
-}
-function sub(a, b, k) {
-  var result = a - b;
-  return k(result);
-}
-function mul(a, b, k) {
-  var result = a * b;
-  return k(result);
-}
-function div(a, b, k) {
-  var result = a / b;
-  return k(result);
-}
-function arithmetic() {
-  add(4, 4, function(a) {
-    sub(a, 2, function(b) {
-      mul(b, 3, function(c) {
-        div(c, 2, function(d) {
-          console.log(d);
-        });
-      });
-    });
-  });
-}
+function add(a,b,k){var result=a+b;return k(result)}function sub(a,b,k){var result=a-b;return k(result)}function mul(a,b,k){var result=a*b;return k(result)}function div(a,b,k){var result=a/b;return k(result)}function arithmetic(){
```

Comment:
Oh, maybe you are reverting it to how it's supposed to be for mapping? If so, sounds good.